samples/drivers/i2s/i2s_codec/boards/mimxrt595_evk_mimxrt595s_cm33.overlay

```diff
@@ -0,0 +1,24 @@
+/ {
+	aliases {
+		i2s-codec-rx = &i2s0;
+		i2s-codec-tx = &i2s1;
+	};
+};
+
+&i3c0 {
+	clk-divider = <20>;
+
```

Comment:
blank line

Reply:
@butok Intentional - I like to semantically separate blocks of code.